src/pharmpy/tools/resmod/tool.py

```diff
@@ -121,6 +126,33 @@ def _create_power_model(input_model):
     return model
 
 
+def _create_combined_model(input_model):
```

Comment:
This is complicating things unnecessarily. No need to be general here. There is only one assignment in the model. Add a new epsilon / IPRED to that expression.

Reply:
Something like:

```
s = model.statements[0]
s.expression = s.expression + new_ruv / IPRED
```
The init for the original sigma can be changed:

```
model.parameters['sigma'].init = new_value
```

The new epsilon and sigma needs to be added the way you do it.

Reply:
Hi Rikard, no problem, I can change it. But I still have a question, do we need to specify the proportional sigma and additive sigma so that we can use the final estimates of these two sigmas to be initial values of the transformed best model. If we need, can I change the original sigma's name from 'sigma' to 'sigma_prop'? I don't know how to change the name and symbol of the original sigma so I delete it and add two new sigma/epsilon.

Reply:
Good point. I think your new version is good.